Blockzilla/TelemetryIntegration.swift

```diff
@@ -49,4 +53,6 @@ class TelemetryEventObject {
     public static let websiteLink = "website_link"
     public static let autofill = "autofill"
     public static let trackerStatsShareButton = "tracker_stats_share_button"
+    public static let quickAddCustomDomainButton = "quick_add_custom_domain_button"
+    public static let requestDesktop = "request_desktop"
```

Comment:
Nit: In the review request (and documentation) you should use the exact string rather than a description, i.e., request_desktop rather than "request desktop". The documentation should basically list all possible values (in this case it's a single string) so it's unambiguous what data is being collected. In the case of dynamic values (e.g., cumulative number of clicks) where you can't really enumerate all the values, you should use a description.